django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -144,17 +166,18 @@
 }
 
 #changelist-filter a {
+    display: block;
     color: #999;
 }
 
 #changelist-filter a:hover {
-    color: #036;
+    color: #036 !important;
```

Comment:
There's a ticket about removing !important from the CSS. In the meantime, it would be nice not to introduce more, if possible.